cli/import-jdl.js

```diff
@@ -440,6 +440,21 @@ class JDLProcessor {
             return [baseName, { clientFramework, serverPort, applicationIndex, devServerPort }];
           })
         );
+
+        const gatewayClientFramework = applicationWithEntities.config.clientFramework;
+        const usedClientFrameworks = relatedApplications
+          .map(([_baseName, { clientFramework }]) => clientFramework)
+          .filter(value => value !== undefined && value !== 'no');
+
+        const differentClientFrameworksUsed = !usedClientFrameworks.every(framework => gatewayClientFramework === framework);
+
+        if (differentClientFrameworksUsed) {
+          throw Error(
+            `Using different client frameworks in microfrontends is not supported. Tried to use: ${gatewayClientFramework} with ${usedClientFrameworks.join(
+              ', '
+            )}`
+          );
+        }
```

Comment:
Please reuse the other loop.
Comparing clientFramework from the microservice with the gateway is enough.
No need to extract every value for the log.

Reply:
> eslint seems to not like it.

Should be forEach instead of map.